astroquery/solarsystem/pds/tests/test_pds.py

```diff
@@ -0,0 +1,219 @@
+DATA_FILES = {'Uranus': 'uranus_ephemeris.html',
+              'Pluto': 'pluto_ephemeris.html',
+              'Neptune': 'neptune_ephemeris.html',
+              'Saturn': 'saturn_ephemeris.html',
+              }
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), "data")
+    return os.path.join(data_dir, filename)
+
+
+# monkeypatch replacement request function
+def nonremote_request(self, request_type, url, **kwargs):
+
+    planet_name = kwargs['params']['center_body']
+    with open(data_path(DATA_FILES[planet_name.capitalize()]), "rb") as f:
+        response = MockResponse(content=f.read(), url=url)
+
+    return response
+
+
+# use a pytest fixture to create a dummy 'requests.get' function,
+# that mocks(monkeypatches) the actual 'requests.get' function:
+@pytest.fixture
+def patch_request(request):
+    mp = request.getfixturevalue("monkeypatch")
+
+    mp.setattr(pds.core.RingNodeClass, "_request", nonremote_request)
+    return mp
+
+
+# --------------------------------- actual test functions
+
+def test_parse_result(patch_request):
+    q = pds.RingNode()
+    # need _last_query to be defined
+    q._last_query = AstroQuery('GET', 'http://dummy')
+    with pytest.raises(ValueError):
+        q.ephemeris('dummy-planet-name')
+
+
+def test_ephemeris_query_Uranus(patch_request):
+
+    pds_inst = pds.RingNode()
+    pds_inst._last_query = AstroQuery('GET', 'http://dummy')
+    bodytable, ringtable = pds_inst.ephemeris(
+        planet="Uranus",
+        epoch="2022-05-03 00:00",
+        location=(-120.355 * u.deg, 10.0 * u.deg, 1000 * u.m),
+    )
+    # check system table
+    systemtable = bodytable.meta
+    assert np.allclose(
+        [-56.12233, -56.13586, -56.13586, -56.01577, 0.10924, 354.11072, 354.12204, 2947896667.0, 3098568884.0, 10335.713263, ],
+        [systemtable["sub_sun_lat"].to(u.deg).value, systemtable["sub_sun_lat_min"].to(u.deg).value, systemtable["sub_sun_lat_max"].to(u.deg).value, systemtable["opening_angle"].to(u.deg).value, systemtable["phase_angle"].to(u.deg).value, systemtable["sub_sun_lon"].to(u.deg).value, systemtable["sub_obs_lon"].to(u.deg).value, systemtable["d_sun"].to(u.km).value, systemtable["d_obs"].to(u.km).value, systemtable["light_time"].to(u.second).value, ],
+        rtol=1e-2,
+    )
+
+    # check a moon in body table
+    mab = bodytable[bodytable.loc_indices["Mab"]]
+    assert mab["NAIF ID"] == 726
+    assert mab["Body"] == "Mab"
+    assert np.allclose(
+        [42.011201, 15.801323, 5.368, 0.623, 223.976, 55.906, 223.969, 56.013, 0.10932, 3098.514, ],
+        [mab["RA (deg)"].to(u.deg).value, mab["Dec (deg)"].to(u.deg).value, mab["dRA"].to(u.arcsec).value, mab["dDec"].to(u.arcsec).value, mab["sub_obs_lon"].to(u.deg).value, mab["sub_obs_lat"].to(u.deg).value, mab["sub_sun_lon"].to(u.deg).value, mab["sub_sun_lat"].to(u.deg).value, mab["phase"].to(u.deg).value, mab["distance"].to(u.km * 1e6).value, ],
+        rtol=1e-2,
+    )
```

Comment:
I'd recommend using a separate `assert` for each column of the table. If this test should fail then with the way it is written currently written it would be quite difficult to figure out which column would be causing the problem.